test/unit/controllers/ApplicationControllerSpec.scala

```diff
@@ -795,6 +835,49 @@ class ApplicationControllerSpec extends UnitSpec with MockitoSugar with WithFake
       }
     }
 
+    "manageSubscription" when {
+      "the user is a superuser" should {
+        "fetch the subscriptions with the fields" in new Setup {
+
+          val subscription = Subscription("name", "serviceName", "context", Seq())
+          givenASuccessfulSuperUserLogin
+          givenTheAppWillBeReturned()
+          given(mockApplicationService.fetchApplicationSubscriptions(any[Application], any[Boolean])(any[HeaderCarrier])).willReturn(Seq(subscription))
+
+          val result = await(addToken(underTest.manageSubscription(applicationId))(aSuperUserLoggedInRequest))
+
+          status(result) shouldBe 200
+          verify(mockApplicationService, times(1)).fetchApplicationSubscriptions(eqTo(application.application), eqTo(true))(any[HeaderCarrier])
+        }
+      }
+
+      "the user is not a superuser" should {
+        "show 401 forbidden" in new Setup {
+          givenASuccessfulLogin()
```

Comment:
401 is unauthorized, 403 is forbidden.  I would expect that in this case, as it is an authorised user lacking in permissions, it should be 403 forbidden.

Reply:
Also - I think it's nicer to check with the FORBIDDEN, OK, SEE_OTHER etc. values rather than the number.

Reply:
I agree with using the words instead of numbers. I think the Unauthorised instead of Forbidden response is due to a method utilised fairly widely (requiresRole) it would extend the scope of this ticket to amend it. I'll add it to tech debt.